src/utils/logger.ts

```diff
@@ -1,5 +1,6 @@
 import * as loggerConfiguration from "../../resources/logger-configuration.json";
-import {PolarisGraphQLLogger} from "@enigmatis/polaris-core";
+import {PolarisLogger} from "@enigmatis/polaris-core";
```

Comment:
Why not `PolarisGraphQLLogger`?

Reply:
Create Polaris connection not works with Polaris graph logger

Reply:
`CreatePolarisConnection` should accept `PolarisGraphQLLogger` because it extends `AbstractPolarisLogger`

[Check here](https://github.com/Enigmatis/polaris-typeorm/blob/master/src/typeorm-bypasses/create-polaris-connection.ts#L12)